### DISCUSSION TITLE
Update feature not working

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#1567] [General]
So, I went to click on updates on ghelper. it was showing my driver was outdated. But it's not, I clicked to download it and was the same driver I already have installed.

**Comment by seerge** [comment]
> ## NOTE
> Bug reports without clear information or scenario to reproduce and logs from ``%AppData%\GHelper`` will be closed without answer. 
Please respect the time of the developer. Thanks.

**Reply by ghost** [reply]
Ill send pics of proof.
